Misc/NEWS.d/next/Library/2019-09-09-04-43-26.bpo-34037.kX3ZqZ.rst

```diff
@@ -0,0 +1 @@
+For :mod:`asyncio`, add attribute ``asyncio.BaseEventLoop.wait_executor_on_close`` to provide an option for waiting for the executor before closing the event loop. Also adds methods ``loop.get_wait_executor_on_close`` and ``loop.set_wait_executor_on_close``.
```

Comment:
I'll fix the number of characters per line and add the appropriate Sphinx roles after I finish adding the documentation changes.